documentation/gnmi/design/dash-schema-relationships.md

```diff
@@ -0,0 +1,25 @@
+# DASH Schema Relationships
+The [figure below](#schema_relationships) illustrates the various schema and their transformations into the various SONiC "domains," including:
+* gNMI northbound API, which useds YANG to specify schema
+* Redis APP_DB and DASH_APP_DB, which use [ABNF](https://github.com/Azure/SONiC/blob/master/doc/mgmt/Management%20Framework.md#12-design-overview) schema definition language
+* [sonic-cfggen](https://github.com/Azure/sonic-buildimage/blob/master/src/sonic-config-engine/sonic-cfggen) JSON and YAML import/export formats
+* [SAI](https://github.com/Azure/DASH/tree/main/SAI) table and attribute objects
+
+All the schema formats are just different ways to represent the same "intent" and cater to the domain they're used in.
+
+* The gNMI, sonic-cfggen and [DASH]_APP_DB schema are all equivalent and can be translated direclty from one to the other without having to change fundamental object structures or quantities.
+
+* The ASIC-DB and SAI table/attribute objects are likewise "equivalent" and differ only in their representation.
+
+* The `orchagent` and `dashorch` perform object transformations; for example, a single entry in the APP_DB can result in multiple ASIC_DB objects with specific relationships. It also performs necessary sequencing and other business logic.
+
+# Canonical Test Data
+
+### Figure - Schema Relationships
+
```

Comment:
I like this diagram and the relationships it shows. Perhaps, it belongs in @prsunny document [SONiC-DASH HLD](https://github.com/Azure/DASH/blob/main/documentation/general/design/dash-sonic-hld.md) when the details are ironed out which is the goal of the PR  https://github.com/Azure/DASH/pull/93.

Reply:
Thank you Michael, that would be a good outcome. I needed to put it somewhere for review. I'm perfectly fine with the content being adapted/merged into https://github.com/Azure/DASH/pull/93 if it's useful, and I can close this PR. Or, I can delete the .md file from this PR, the image can remain and accepted into /images, then referred to in https://github.com/Azure/DASH/pull/93.

Reply:
I suggest keeping the 2 PRs open for now. 
When we meet next time, and after some feedback hopefully, we can merge them.

Reply:
Agree with @mmiele, this content is more on Sonic. You can update the Sonic HLD to have this details captured and  update the diagram.

Reply:
@prsunny would you please review again, I merged this into your dash-sonic-hld. I still need some clarity on the final JSON config format. Thanks.